src/uct/tcp/tcp_cm.c

```diff
@@ -597,7 +599,13 @@ uct_tcp_cm_handle_conn_req(uct_tcp_ep_t **ep_p,
             goto out_destroy_ep;
         }
     } else {
-        ucs_assert(uct_tcp_cm_ep_accept_conn(ep));
+        ucs_assertv(uct_tcp_cm_ep_accept_conn(ep),
```

Comment:
code inside uct_tcp_cm_ep_accept_conn looks very similar? maybe unite and print more info such as conn_state and result of compare?

Reply:
yes but `cmp < 0` is not always bad case, printing message to a buffer on any result of compare will add extra overhead for good case. I added a func to reuse the code